src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1637,6 +1639,70 @@ private BoundExpression BindCastCore(ExpressionSyntax node, BoundExpression oper
             return CreateConversion(node, operand, conversion, isCast: true, wasCompilerGenerated: wasCompilerGenerated, destination: targetType, diagnostics: diagnostics);
         }
 
+        private BoundExpression BindOriginal(OriginalExpressionSyntax syntax, DiagnosticBag diagnostics)
+        {
+            var containingMethod = (MethodSymbol)this.ContainingMember();
+            var symbol = containingMethod.AssociatedSymbol ?? containingMethod; // property, event, or non-accessor method
+            var receiver = symbol.IsStatic ? null : ThisReference(syntax, symbol.ContainingType, wasCompilerGenerated: true);
+            var replaced = symbol.Replaced;
+            Debug.Assert((object)replaced != null);
+            Debug.Assert(replaced.Kind == symbol.Kind);
+
+            switch (symbol.Kind)
+            {
+                case SymbolKind.Method:
+                    {
+                        var replacedMethod = (MethodSymbol)replaced;
+                        return new BoundMethodGroup(
+                            syntax,
+                            default(ImmutableArray<TypeSymbol>),
+                            replacedMethod.Name,
+                            ImmutableArray.Create(replacedMethod),
+                            lookupSymbolOpt: replacedMethod,
+                            lookupError: null,
+                            flags: BoundMethodGroupFlags.None,
+                            receiverOpt: receiver,
+                            resultKind: LookupResultKind.Viable);
```

Comment:
The replaced method is no longer virtual. Regardless, the call doesn't need to be virtual so emit now generates `Call` rather than `CallVirt` for replaced methods.